apps/game_backend/lib/game_backend/units/characters/character.ex

```diff
@@ -30,11 +30,9 @@ defmodule GameBackend.Units.Characters.Character do
     field(:base_speed, :float)
     field(:base_size, :float)
 
-    # TODO This should be removed once we have the skills relationship, issue: https://github.com/lambdaclass/mirra_backend/issues/717
-    field(:skills, {:map, :string})
-
     belongs_to(:basic_skill, Skill, on_replace: :update)
     belongs_to(:ultimate_skill, Skill, on_replace: :update)
+    belongs_to(:dash_skill, Skill)
```

Comment:
Don't we need to add `on_replace: :update` here too?

Reply:
Yep. Done.